concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-322	322	Martello	hammer	1481	HAMMER	700	697	595	209	653	106	275
+DellaRosa-2010-417-323	323	Matita	pencil	2578	PENCIL	694	684	679	194	694	100	121
+DellaRosa-2010-417-324	324	Matrimonio	wedding	3182	MARRIAGE OR WEDDING	489	621	616	248	632	313	375
+DellaRosa-2010-417-325	325	Mela	apple	1320	APPLE	694	676	670	156	694	100	121
+DellaRosa-2010-417-326	326	Melanzana	eggplant	1146	AUBERGINE	700	695	545	275	600	100	208
+DellaRosa-2010-417-327	327	Mendicante	beggar	1934	BEGGAR	637	629	405	333	574	163	300
+DellaRosa-2010-417-328	328	Minerale	mineral			627	538	447	396	505	174	388
+DellaRosa-2010-417-329	329	Mucca	cow	1007	COW	691	662	618	163	680	100	206
+DellaRosa-2010-417-330	330	Muscolo	muscle	1004	MUSCLE	623	635	616	344	588	156	319
+DellaRosa-2010-417-331	331	Musica	music	642	MUSIC	473	459	676	188	630	291	200
+DellaRosa-2010-417-332	332	Naso	nose	1221	NOSE	688	680	658	141	688	100	116
+DellaRosa-2010-417-333	333	Negozio	shop	752	STORE	668	648	658	221	656	125	163
+DellaRosa-2010-417-334	334	Noce	walnut	2042	WALNUT	694	668	591	235	638	105	147
+DellaRosa-2010-417-335	335	Nomade	nomad			588	548	339	418	544	190	416
+DellaRosa-2010-417-336	336	Nonno	grandfather	1383	GRANDFATHER	674	685	689	118	647	125	350
+DellaRosa-2010-417-337	337	Notte	night	1233	NIGHT	571	660	688	163	665	221	130
+DellaRosa-2010-417-338	338	Oca	goose	1187	GOOSE	685	700	468	150	659	106	197
+DellaRosa-2010-417-339	339	Oceano	ocean	645	OCEAN	594	642	555	295	594	165	396
+DellaRosa-2010-417-340	340	Ombrello	umbrella	3105	UMBRELLA	677	700	647	175	688	103	128
+DellaRosa-2010-417-341	341	Orologio	clock	1323	CLOCK	688	647	659	210	624	118	144
+DellaRosa-2010-417-342	342	Orso	bear	1820	BEAR	659	700	487	225	642	110	265
+DellaRosa-2010-417-343	343	Ospedale	hospital	616	HOSPITAL	658	693	656	241	696	167	270
+DellaRosa-2010-417-344	344	Ospite	guest	1782	GUEST	550	497	594	322	580	221	338
+DellaRosa-2010-417-345	345	Palma	palm			679	688	487	311	650	110	244
+DellaRosa-2010-417-346	346	Palude	swamp	1145	SWAMP	645	613	332	341	578	175	310
+DellaRosa-2010-417-347	347	Pantaloni	trousers	809	TROUSERS	658	696	684	144	682	115	125
+DellaRosa-2010-417-348	348	Pappagallo	parrot	882	PARROT	686	668	541	196	620	100	244
+DellaRosa-2010-417-349	349	Passeggero	passenger			625	574	536	305	594	185	372
+DellaRosa-2010-417-350	350	Pecora	sheep	1331	SHEEP	695	694	595	173	626	113	413
+DellaRosa-2010-417-351	351	Pelle	skin	763	SKIN	668	656	667	235	638	129	159
+DellaRosa-2010-417-352	352	Pennello	brush	3094	PAINTBRUSH	691	665	547	229	635	106	256
+DellaRosa-2010-417-353	353	Peperone	pepper			700	697	647	218	647	106	300
+DellaRosa-2010-417-354	354	Pera	pear	922	PEAR	694	684	627	153	694	100	132
+DellaRosa-2010-417-355	355	Perla	pearl	3325	PEARL	687	680	497	331	613	129	225
+DellaRosa-2010-417-356	356	Persone	people	789	PEOPLE	611	644	663	173	615	200	300
+DellaRosa-2010-417-357	357	Pescatore	fisherman	1858	FISHERMAN	661	673	509	244	626	142	215
+DellaRosa-2010-417-358	358	Pianeta	planet			563	663	482	300	547	241	444
```

Comment:
Maybe a candidate for a new concept? Already a few instances in previous lists.

Reply:
@MiraAhmedovic Could you open an issue linking this PR where we can decide whether we want to include PLANET as a new concept?